raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -11,30 +11,29 @@ library ChannelManagerLibrary {
         Token token;
     }
 
-    /// @notice getChannelsAddresses to get all channels
+    /// @notice Get all channels
     /// @dev Get all channels
-    /// @return channels (address[]) all the channels
+    /// @return all the channels
     function getChannelsAddresses(Data storage self) returns (address[] channels) {
         channels = self.all_channels;
     }
 
-    /// @notice getChannelsForNode(address) to get channels that 
-    /// the address participates in.
+    /// @notice to get channels that the address participates in.
     /// @dev Get channels where the given address participates.
     /// @param node_address (address) the address of the node
-    /// @return (address[]) of the channel's addresses that node_address participates.
+    /// @return of the channel's addresses that node_address participates.
```

Comment:
This does not make a lot of sense. Maybe something like `return The channels that the node_address participates in`